src/domains/dashboard/components/node-view/node-view.tsx

```diff
@@ -144,16 +145,27 @@ export const NodeView = ({
   children,
   nodeIDs,
   reportEvent = noop,
+  defaultChart = "",
 }: Props) => {
   const [width, setWidth] = useState(0)
-  const [currentChart, setCurrentChart] = useState("")
+  const [currentChart, setCurrentChart] = useState(defaultChart)
+
   const ref = useRef<HTMLDivElement>(null)
   useEffect(() => {
     if (!width && ref.current) {
       setWidth(ref.current.getBoundingClientRect().width)
     }
   }, [width])
 
+  useEffect(() => {
+    if (!defaultChart) return
+
+    setTimeout(() => {
+      const chartElement = document.querySelector(`#${defaultChart}`)
+      if (chartElement) chartElement.scrollIntoView()
+    }, 100)
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
```

Comment:
It would be better to clear the timeout id on unmount.
Are we sure that there is not any safer way to ensure that the charts have been mounted? Maybe an `onReady` callback?

Reply:
does `0` works the same as `100`? `0` is still a timeout-hack, but at least it's safer against race-conditions :)